src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -231,7 +231,7 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
         compound.setTag(TAG_ASYNC_REQUESTS, getAsyncRequests().stream().map(StandardFactoryController.getInstance()::serialize).collect(NBTUtils.toNBTTagList()));
     }
 
-    public Set<IToken> getAsyncRequests()
+    public Set<IToken<?>> getAsyncRequests()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)